### ISSUE TITLE
consistency loss and entropy loss

### ISSUE THREAD

**Issue by ipheiman** [wvangansbeke/Unsupervised-Classification#82]
Hi authors,
Thank you for the brilliant work!
I have tried training step 2: SCAN Clustering on my custom dataset but the consistency loss started at around 3.0 and after 300 epochs, it only decreased to 1.9, whereas entropy loss stayed relatively the same. I was wondering for the imagenet subsets training, what was the consistency and entropy loss obtained for your training.

Thanks for reading this! Cheers :)
